ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/ReplaceWithFSharpOperatorFix.fs

```diff
@@ -0,0 +1,36 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Resources.Shell
+
+type ReplaceWithFSharpOperatorFix(error: ConstraintAddingError) =
+    inherit FSharpQuickFixBase()
+
+    let ref = error.Expr.As<IReferenceExpr>()
+    let fsharpOperator =
+        match ref with
+        | null -> ""
+        | ref ->
+
+        match ref.ShortName with
+        | "==" -> "="
+        | "!=" -> "<>"
+        | _ -> ""
+
+    override x.IsAvailable _ = isValid ref && fsharpOperator <> ""
```

Comment:
Please check that the reference is in a binary app expression.
Also, there seems to be no test scenario like this: <img width="723" alt="Screenshot 2021-03-01 at 18 10 49" src="https://user-images.githubusercontent.com/3923587/109516650-770fe180-7ab9-11eb-86ed-438067b44939.png">

Reply:
I think the check for IBinaryAppExpr is unnecessary because we can apply the fix to more cases such as 
`fun x -> (==) 1`

Reply:
Then we should take into account possible qualifier expressions and/or type argument lists, so setting the reference name only is likely not enought.

Reply:
Also, if we're to support prefix app expression cases, please add a test case. 🙂